src/components/Editor/Tabs.css

```diff
@@ -108,13 +105,9 @@
 }
 
 .source-tab .close-btn {
```

Comment:
rules are now merged, but had to explicitly write the rule `html[dir="ltr"] .source-tab:hover .close-btn`